Resources/config/templating_php.xml

```diff
@@ -17,7 +17,7 @@
             <argument type="service" id="assetic.asset_factory" />
         </service>
 
-        <service id="assetic.helper.static" class="%assetic.helper.static.class%" scope="request">
+        <service id="assetic.helper.static" class="%assetic.helper.static.class%">
             <tag name="assetic.templating.php" />
             <argument type="service" id="templating.helper.assets" />
```

Comment:
``templating.helper.assets`` can be request-scoped in older version of Symfony, so you need to check whether it is the case and scope our own helper in such case